homeassistant/components/mqtt/quality_scale.yaml

```diff
@@ -0,0 +1,122 @@
+rules:
+  # Bronze
+  action-setup: done
```

Comment:
I believe the `keepalive` field appears to be missing a `data_description`

Reply:
Thanks, I'll open a PR for that.

Reply:
https://github.com/home-assistant/core/pull/131446